docs/index.rst

```diff
@@ -44,15 +44,20 @@ Small sample script that shows how to get started with RedisCluster. It can also
     >>> # Requires at least one node for cluster discovery. Multiple nodes is recommended.
     >>> startup_nodes = [{"host": "127.0.0.1", "port": "7000"}]
 
-    >>> # Note: decode_responses must be set to True when used with python3
+    >>> # Note: decode_responses gives different results with python3 and python2
```

Comment:
I agree that removing the `must be set to True...` is good to remove, but i am not sold on the change to `gives different results...` that is very ambiguous and non descriptive. Suggestion is to tell something like `When set to True in python3 it will convert all binary responses into strings` or something similar.